apps/dashboard/src/lib/utils/functions/formatTime.js

```diff
@@ -0,0 +1,9 @@
+export default function formatTime(utcTimestamp) {
+  const date = new Date(utcTimestamp);
+  let hours = date.getUTCHours();
+  const minutes = date.getUTCMinutes();
+  const ampm = hours >= 12 ? 'PM' : 'AM';
+  hours = hours % 12 || 12;
+  const formattedTime = `${hours}:${minutes.toString().padStart(2, '0')}${ampm}`;
+  return formattedTime;
```

Comment:
What's the goal of this function?

There is an implementation of how we use `dayjs`, let me know your use case and I'll properly direct you.

Reply:
the created_at timestamp from the database is converted to a 12hr time format here,i couldnt find any pre existing function in the utils so i wrote this

Reply:
right now it just indicates the formatted created _at time like 9:33 AM but something i intend to do is such that it displays yesterday or the full date it was sent like 11 feb 12:00 AM, when the feed is an old feed

Reply:
the dayjs formatDate function in the utils file only returns today, yesterday

Reply:
i implemented it using dayjs,i read the docs and the format date code and figured how to do it,a dayjs inbuilt plugin called localizedFormat was used 
![Screenshot (716)](https://github.com/rotimi-best/classroomio/assets/69228399/92c10fb1-cbb1-49eb-98cc-395602030d57)

i am trying to send a screenshot but its getting broken

Reply:
its probably because i am using airtel

Reply:
Use this function `calDateDiff`.

<img width="796" alt="image" src="https://github.com/rotimi-best/classroomio/assets/31730715/e56b5310-bd46-4e83-b684-9d9d3191973a">

We already use it on the community question page and many other places.
<img width="1217" alt="image" src="https://github.com/rotimi-best/classroomio/assets/31730715/ec9493d1-40f7-4055-98d5-36ad556d660f">

Reply:
done